.github/workflows/javascript-test.yml

```diff
@@ -0,0 +1,21 @@
+name: Continuous testing
+
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use node 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Run tests
+        run: yarn test
+      - name: Run lint
+        run: yarn lint
+      - name: Build the project
+        run: yarn build
```

Comment:
I would put `build` before `tests` - if the project doesn't build, it probably won't pass the tests.